install/yaml/install.yaml

```diff
@@ -125,6 +125,23 @@ metadata:
 ---
+# Source: agones/templates/service/allocation.yaml
+# Default allocation client secret
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: allocator-client.default
+  namespace: default
+  labels:
+    app: agones-manual
+    chart: "agones-1.8.0-dev"
+    release: "agones-manual"
+    heritage: "Helm"
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5VENDQWJHZ0F3SUJBZ0lRVmJVdGxSV3huVDBBT1lGVC9NbzhRREFOQmdrcWhraUc5dzBCQVFzRkFEQUEKTUI0WERUSXdNRGN5TVRBek1UWXpOMW9YRFRNd01EY3hPVEF6TVRZek4xb3dBRENDQVNJd0RRWUpLb1pJaHZjTgpBUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTiszdHZMdlhsNVhneXdpSFgyOGw3K2Z3TUJHUTF3alRSdlNDdHF1CnQwaDZvUEtzRksydEZaQUlSWnFpbVVYL1UyR0pzWURqRUJRWVFTVSs1WVp0cTFZbGtVQmFEOUl5ZnEzNlNiNWcKUWw2eUFlU3AvblFPS3JoWC8zdFVwK1RnZ3RXcEpaTlZlSWlhM0kwVG1TbVFreTFnZmpGblMva01JdHFDZ2k2ZwpzRUlyNlFvVktEdHBzVFVHdXU3Wk55OHk4dHYvRjZObGZQelJpcGRkS3lyaFo2elhvaG9QSHRqSFM0TlhRMmxhCkdkQ3hveTIzV2VFODV0eHoyRWRSZkt0TXNWSmZmbjA3bEx0SDlhVnplcHduVzQzcG9jaE1zM1ZweFdhUW5JdjQKZ05yNGNHZ2thbm9oNDA5WUhpZkNnNkwyNFhVSUY0azVxNEpTcGZtY2h6RFFldDBDQXdFQUFhTS9NRDB3RGdZRApWUjBQQVFIL0JBUURBZ1dnTUIwR0ExVWRKUVFXTUJRR0NDc0dBUVVGQndNQkJnZ3JCZ0VGQlFjREFqQU1CZ05WCkhSTUJBZjhFQWpBQU1BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQUQ5VXIxY2dnSVVvMVExV1lIM0IycnROTVMKeHNPTGc3YXBaTE92cUdWWmxmMFg4MFhBMFlQVE5jaEFTdExVbU5TVHNkanQ3ZFJiZjI0M29saS8rTXNIQ25nQgprY1crajR3NjNvalU3M1E3TEJNSGZYSDMwYmRKRE9MNStvVnhnbUgyNmVQVEFhQXBLWXlQem1OblZubzFyOUNpCk51Y2QxK0NsSmx0ZjFmRi90LzBoSGpwV2REbGpRUktOQjd6THlzV25MdTN2UW9iSTBZK3E1TXgvNUQvb2xyeHoKQkpWR1pPRUgyK0pQQy8rb0lCMWFzSmJJcW9vemNmbEJ1SWY0d21sN3hxWVg2YWxPOXZqT2d1UEdYb05Pc3A1WQo0WUZyRHdCaXUrUkZ6Y2VtNmR3eE5yOUJNSFQ0cFcxU2g0M2REWkdKZDBPbnJ4VjdtT1pmQmo1cWNhekIKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
Set the helm config to false to avoid generating this every time. Then you need to have a client cert stored [here](https://github.com/googleforgames/agones/tree/master/install/helm/agones/certs/allocator) to be picked up by helm.

Reply:
Done.